sdk/storage/azure-storage-blobs/CHANGELOG.md

```diff
@@ -1,10 +1,25 @@
 # Release History
 
-## 1.0.0-beta.5 (Unreleased)
+## 12.0.0-beta.5 (2020-11-13)
+
+### New Features
+
+* Support for replaceable HTTP transport layer.
+* Add `version.hpp`.
 
 ### Breaking Changes
```

Comment:
I am surprised, but did we really have no bug fixes reported or made during this release? I don't see any myself, but maybe I missed it, so just want to confirm.

Reply:
no, i'm not really good at writing bugs.